AWSiOSSDKv2.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the use of this file?

Reply:
This was an inadvertent change and can be deleted.